hugo.toml

```diff
@@ -509,3 +509,18 @@ time_format_blog = "02.01.2006"
 language_alternatives = ["en"]
 description = "Довершена система оркестрації контейнерів"
 languageNameLatinScript = "Ukrainian"
+
+[languages.ar]
+title = "كوبرنيتيز"
+languageName = "العربية (Arabic)"
+weight = 17
```

Comment:
I think the weight should be different at some point; I'd sort languages alphabetically by their English equivalents.